chapters/1.md

```diff
@@ -11,31 +11,31 @@ Maria is purchasing a car from George, and in this chapter we will consider thre
+Banks limit your access to your money. They may be closed on weekends and when there is large emergencies or during holidays (which vary from country to country). If the bank's servers are shut down for maintenance or there is a security issue in the e-banking system or application, customers have no recourse but to wait until it is made available again. This can lead to significant delays, frustration and loss of economic opportunities by those who rely on the banks.
 
-Moreover, encrypting and undisclosed algorithms may predict if you can be considered one potential thief. They work based on a undescribed amount of personal data crossed with e-shops informations, time, and many others such as habits and even your entire digital life. While I am not totally against having the security of the nation or the system as the justification of the surveillance, the algorithm methods could mark an account as malicious and censorship that accounts or those funds. Have you moved $1,000 dollars in a foreign country for your holiday? I think it could be unpleasant to know that your payment has been refused.
+Moreover, opaque monitoring systems are used to identify potentially fraudulent behavior. These systems utilize data sources far exceeding the information you shared with your bank including online purchases, location, credit info, social media and many other aspects of your private life, both online and offline. General patterns of expected behavior are used to identify and even prevent actions by the customer deemed unusual.
 
-In addition to this, we have also to think about the emission of money and who controls it. At the moment, almost all the economies are based on central banks managed by private entities (such as Federal Reserve in United States) or big corporations (e.g. ECB – Europe Central Bank) ruled by governments. These adopt acts, agreements between states that formally decided how much banknotes the national bank should print. Another issue of the actual monetary system is that no one - including the companies, authories and governements - can know how much American dollars or Euros are circulating at the moment. Even if the centralized banks are trying to be transparent in papers reporting information, there is no way to check those datas.
+Such systems can lead to false positives (potentially being treated like a criminal because of a system which is supposed to protect you from criminals), increased corruption by those in privileged positions and many subtle forms of economic censorship. One might also experience more mundane annoyances like having an ATM card refused while on vacation, payments being blocked, losing access to an e-banking account or unwanted delays due to any behavior flagged by their fraud system. There is also significant amounts of private data shared with third party "partners" and various government surveillance systems.
```

Comment:
>  There is also significant amounts of private data shared with third party "partners" and various government surveillance systems.

In this section, we do not mention the problems related to privacy.

Reply:
Removed.